dask_cuda/benchmarks/local_cupy_map_overlap.py

```diff
@@ -148,11 +134,25 @@ async def run(args):
                 print_key_value(key="NVLink", value=f"{args.enable_nvlink}")
             print_key_value(key="Worker thread(s)", value=f"{args.threads_per_worker}")
             print_separator(separator="=")
-            print_key_value(key="Wall clock", value="npartitions")
+            print_key_value(key="Wall clock", value="Partitions")
             print_separator(separator="-")
-            for (took, npartitions) in took_list:
-                t = format_time(took)
-                print_key_value(key=f"{t}", value=f"{npartitions}")
+            times = []
+            for idx, (took, npartitions) in enumerate(took_list):
```

Comment:
`idx` unused

Reply:
Done in https://github.com/rapidsai/dask-cuda/commit/dc1a58715b0053eac7662229e8b892707a7d82e3